modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,40 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+
+[#release-703]
+== Release 7.0.3 (December 2021)
+Couchbase Server 7.0.3 was released in December 2021. 
+This maintenance release contains an important security fix.
+
+[#fixed-issues-703]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
+
+[IMPORTANT]
+====
+This release fixes a high severity vulnerability related to Log4j 2. 
+All users should upgrade to this release as soon as possible.
```

Comment:
Can we change this to
All users utilising the Couchbase Analytics service should upgrade to this release as soon as possible.